### DISCUSSION TITLE
Single Oathkeeper as API for multiple sub-domains

### DISCUSSION THREAD

**Discussion by WilliamsDL** [ory/oathkeeper#1101] [Q&A]
How do I configure Oathkeeper API to match paths to multiple sub domains?

abc.domain.com/v1/path1
abccc.domain.com/v1/path1. (note the same path, different subdomain)
bbc.domain.com/testpath
ccc.domain.com/something

And Oathkeeper itself will not be on any of these subdomains.  It will be a single instance, running inside Kubernetes with a K8s Service to allow access.

How do I configure the rules to match in this scenario?

**Comment by WilliamsDL** [comment]
Is the intended solution to define the match like this?
      match:
        url: http://oathkeeper-api.local.com/https://abccc.comain.com/v1/path1/<.*>

Doing this, I'm able to make my use-case work, but it seems wonky.

**Reply by vinckr** [reply]
Hello @WilliamsDL 
I think this is correct, what does seem wonky?
Related [SA](https://stackoverflow.com/questions/68784760/ory-oathkeeper-match-multiple-rules) & [Blogpost](https://gruchalski.com/posts/2021-05-20-figuring-out-oathkeeper/)
